### ISSUE TITLE
Chinese Translation

### ISSUE THREAD

**Issue by pengchzn** [labmlai/annotated_deep_learning_paper_implementations#257]
I noticed that the Chinese translation appears to be machine-generated. Do you have any plans to officially translate this excellent project? If needed, I am willing to take on the responsibility for the Chinese translation!

**Comment by vpj** [comment]
Yeah it's machine generated. We cannot translate because we don't know Chinese unfortunately. It's something that'll be quite useful since most of the readers are Chinese speaking.

The translations are in the folder https://github.com/labmlai/annotated_deep_learning_paper_implementations/tree/master/translate_cache

These are then used to generate the HTML

**Comment by pengchzn** [comment]
Got it. So I just need to modify the translations in the `translate_cache` folder, right? Also, when submitting a PR, should I do it chapter by chapter or wait until the entire translation is finished and submit it all at once?

I am very grateful for the effort everyone has put into this project. It has given me a thorough understanding of neural networks. I hope to contribute to making it better.

**Comment by vpj** [comment]
Lets do PRs and merge them as you translate. It'll be a lot faster and you'll be able to see how the translations appear on nn.labml.ai. May be others will also join you to help.

**Comment by vpj** [comment]
Lets first do on one of the simplest files and test.
